core/src/main/java/io/lionweb/lioncore/java/serialization/JsonSerialization.java

```diff
@@ -496,19 +479,16 @@ private List<ClassifierInstance<?>> deserializeClassifierInstances(
             new MapBasedResolver(deserializedByID), this.instanceResolver);
     serializedClassifierInstances.stream()
         .forEach(
-            n -> {
+            node -> {
```

Comment:
should we call the variable `classifierInstance` or `instance` or `ci`?

Reply:
(see previous comment)